### ISSUE TITLE
Remove Erode / Dilate from Colored Shape or Make them Useful

### ISSUE THREAD

**Issue by jasondaming** [PhotonVision/photonvision#419]
These options seem to do nothing currently.

**Comment by jsimpso81** [comment]
I saw this too.   It appears that the code is currently commented out.  I made some modifications and got it working.  A couple of suggestions.  1) Make these into "open" and "close" operations - These may be more useful.  2) If not added to the user interface, update the kernel size based on screen resolution ( <=320 -- 2, <=640 -- 4, otherwise 5 ),  3) Perhaps implement Open, Close, Open and allow user to select which ones to use so they can select the order.  (Maybe even implement a count 1-3 -- not certain if repeated actions help or not.)
These appear to be fairly efficient in terms of CPU usage and seem to speed up the actions that follow.
I'd be happy to share the code changes if desired.
